packages/components/src/range-control/styles/range-control-styles.js

```diff
@@ -150,28 +151,30 @@ export const ThumbWrapper = styled.span`
 	top: 0;
 	user-select: none;
 
-	[dir=rtl] & {
+	[dir='rtl'] & {
```

Comment:
is it possible to have bable plugin or something that generates the RTL styles like we do with sass?

Reply:
@youknowriad Not at this very moment, although having an automated solution would be lovely!

CSS-in-JS libraries need some way to adjust the CSS compiler output (`stylis`), typically with a Provider of some kind, ([discussion + examples](https://github.com/emotion-js/emotion/issues/793))